driver/Dockerfile

```diff
@@ -1,3 +1,12 @@
+# MULTI-STAGE BUILD for IBM Spectrum Scale CSI Driver
+
+FROM golang:1.13.1 AS builder
+WORKDIR /go/src/github.com/IBM/ibm-spectrum-scale-csi-driver/
```

Comment:
Is this still going against the old repo? or did you clone down the combined repo and give it a different name?
`git clone https://github.com/IBM/ibm-spectrum-scale-csi ibm-spectrum-scale-csi-driver`

Reply:
This is just what we name ourselves inside the docker, shouldn't have any impact otherwise.  I will update though